ci/test_wheel_cudf_polars.sh

```diff
@@ -43,6 +47,9 @@ python -m pip install \
     "$(echo ./dist/libcudf_${RAPIDS_PY_CUDA_SUFFIX}*.whl)" \
     "$(echo ./dist/pylibcudf_${RAPIDS_PY_CUDA_SUFFIX}*.whl)"
 
+rapids-logger "Pin to 1.7.0 Temporarily"
```

Comment:
this can be removed now

Reply:
I would do those changes in a follow-up PR where we unpin and also change the xfail/xpass status of that one test we discussed.